dockerfiles/alpine/Dockerfile

```diff
@@ -0,0 +1,73 @@
+# build image
+FROM rust:1.47.0 as cargo-build
+
+# rust toolchain version
+ARG RUST_TC_VER=nightly-2020-10-17
+
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends musl-tools \
+  && rustup install $RUST_TC_VER \
+  && rustup default $RUST_TC_VER \
+  && rustup target add x86_64-unknown-linux-musl
+
+WORKDIR /usr/src/bob
+
+# crates downloading and initial build
+COPY Cargo.toml Cargo.toml
```

Comment:
Now that the `Dockerfile` is in `dockerfile/alpine/` directory, should the paths for the `COPY` command be updated?

Reply:
Dockerfile is supposed to be run with context set to root dir (as in compose examples)